src/barbequeue/messaging/classes.py

```diff
@@ -28,3 +27,51 @@ def serialize(self):
         ), "Message type not found in predetermined message type list!"
 
         return json.dumps({"type": self.type, "messsage": self.message})
+
+
+class SuccessMessage(Message):
+    def __new__(cls, job_id, result):
+        msg = {'job_id': job_id, 'result': result}
+        self = super(SuccessMessage, cls).__new__(cls, type=MessageType.JOB_COMPLETED, message=msg)
```

Comment:
This seems a bit magic... can the same be achieved through normal constructors, properties and inheriting a type from `namedtuple` ?